Nimble/Matchers/RaisesException.swift

```diff
@@ -10,14 +10,14 @@ import Foundation
 /// nil arguments indicates that the matcher should not attempt to match against
 /// that parameter.
 public func raiseException(
-    named: String? = nil,
+    named named: String? = nil,
```

Comment:
Ah, gotcha. Yeah, open an issue if you feel strongly about it and we'll deal with it separately. :snail: